packages/@aws-cdk/aws-cloudfront/test/web-distribution.test.ts

```diff
@@ -198,7 +218,9 @@ nodeunitShim({
           behaviors: [
             {
               isDefaultBehavior: true,
-              trustedSigners: ['1234'],
```

Comment:
Can you keep this in as well? Would like to make this purely additive.

Reply:
Fixed.